grails-app/conf/BootStrap.groovy

```diff
@@ -104,7 +104,8 @@ class BootStrap {
             [
                     id         : shipmentType.id,
                     name       : shipmentType.name,
-                    description: shipmentType.description
+                    description: shipmentType.description,
+                    label      : shipmentType.displayName,
```

Comment:
I might prefer this just being called displayName since we've already used that for Product. Adding a new `label` property would just add confusion.

Reply:
sure, I can change it. I've done it as `label` to avoid having to do some additional mapping on the frontend side for the `Select` to work fine, but no worries about that.